website/docs/getting-started/pulling-an-image.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+title: Pulling an image
+description: Pulling an image to your container engine.
+keywords: [podman desktop, podman, containers, images]
+tags: [images]
+---
+
+# Pulling an image to your Container engine
+
+With Podman Desktop, you can pull an image from an OCI regitry, to your container engine.
+
+#### Prerequisites
+
+- The image is available in a OCI registry.
+- If the registry or the image are not publicly available, you configured access to the registry on Podman Desktop.
```

Comment:
should be a link there on how to do it?

Reply:
Indeed, but the doc is not there yet. I'll add the link later to keep pull requests small.

Reply:
so maybe, 
`If the registry or the image are not publicly available, access is already granted or configured through the Settings/Registries page of Podman Desktop`